OneSignalExample/Packages/com.onesignal.unity.core/Runtime/Core/OneSignal.cs

```diff
@@ -0,0 +1,1449 @@
+﻿/**
```

Comment:
Needs to be updated to match change with master:
https://github.com/OneSignal/OneSignal-Unity-SDK/blob/master/OneSignalExample/Assets/OneSignal/src/OneSignal.cs